sdk/include/reactos/undocuser.h

```diff
@@ -204,6 +204,20 @@ BOOL WINAPI PaintMenuBar(HWND hWnd, HDC hDC, ULONG left, ULONG right, ULONG top,
 #define DrawCaptionTemp DrawCaptionTempA
 #endif
 
+//
+// Hard error balloon package
+//
+typedef struct _BALLOON_HARD_ERROR_DATA
+{
+    DWORD cbHeaderSize;
+    DWORD Status;
+    DWORD dwResponseType;
+    ULONG_PTR TitleOffset;
+    ULONG_PTR MessageOffset;
+} BALLOON_HARD_ERROR_DATA, *PBALLOON_HARD_ERROR_DATA;
+
+/* The dwData value needed for COPYDATASTRUCT */
+#define BALLOON_DATA 0xc07f
```

Comment:
Nope. Restart your computer completely, then re-do what you have done to obtain this value, and you'll see that it should change. This is because both usersrv and the shell call RegisterWindowMessageW(L"HardError"); to get a unique window message number (unique to this session).